lib/unwrappr/cli.rb

```diff
@@ -62,38 +67,58 @@ def execute
       option(['-r', '--repo'],
              'REPO',
              <<~DESCRIPTION,
-               a repo in github <owner/project>, may be specified multiple times
+               a repo in GitHub <owner/project>, may be specified multiple times
              DESCRIPTION
              required: true,
              multivalued: true)
 
+      option ['-R', '--recursive'],
+             :flag,
+             'Recurse into subdirectories',
+             attribute_name: :recursive
+
       def execute
         repo_list.each do |repo|
-          unless Dir.exist?(repo)
-            GitCommandRunner.clone_repository(
-              "https://github.com/#{repo}",
-              repo
-            )
-          end
+          GitCommandRunner.clone_repository("https://github.com/#{repo}", repo) unless Dir.exist?(repo)
```

Comment:
TIL: the Rubocop default line length has been upped to 120.
https://github.com/rubocop/rubocop/pull/7952